docs/proposals/Lifelong-learning-Ops-ui.md

```diff
@@ -0,0 +1,166 @@
+### goals
+
+- Supports metrics collection and visualization in lifelong learning
+- Support unified management and search of application logs
+- support the management and status monitoring of lifelong learning in dashboard
+## proposal
+We proposal using grafana,loki and prometheus to display the mertic,log and statues about the lifelong learnig job. The Prometheus is used to collect the mertic and status and the loki is used to collect logs.
+### use cases
+
+- Users can search log by specifying pod and keywords in the log control panel
+- User can view the history state or the mertic of any component in a sequence diagram
+- Users can view the status of any CRD (Model, DataSet, lifelongjob) by specifying the name
+## design details
+### Architecture
+![](./images/lifelong-learning-ops-architecture.png)
+### loki
+At present, the log file is under the directory /var/log/contains.
+There are two types of Promtail collection modes. The following table shows the feature comparison:
+
+|                            | Daemonset                 | SideCar                                   |
+|----------------------------|---------------------------|-------------------------------------------|
+| Source                     | Sysout+ part file         | file                                      |
+| Log classification storage | mapped by container/path  | Pod can be separately                     |
+| Multi-tenant isolation     | isolated by configuration | isolated by Pod                           |
+| Resource occupancy         | low                       | high                                      |
+| Customizability            | Low                       | High, each Pod is configured individually |
+| Applicable scene           | Applicable scene          | Large, mixed cluster                      |
+
+### Prometheus
+#### CRD metric
+
+1. LLJob Status
+
+| **Metric**           | **Type** | **Range**                                             |
+|----------------------|----------|-------------------------------------------------------|
+| JobStatus            | gauge    | Enum(True, False, Unknown)                            |  
+| StageConditionStatus | gauge    | Enum(waiting,ready,starting,running,completed,Failed) |  
+| Stage                | gauge    | Enum(Train, Eval, deploy )                            |  
+
+2. Dataset status
+
+| **Metric**       | **Type** | **Range**                           |
+|------------------|----------|-------------------------------------|
+| NumberOfSamples  | gauge    | The number of samples               |
+| StageTrainNumber | gauge    | The number of samples used by train |
+| StageEvalNumber  | gauge    | The number of samples used by eval  |
+
+3. Model
```

Comment:
Lack of algorithm-related metrics, such as metrics to evaluate the overall performance of the lifelong learning paradigm.